Index: ecore-1.7.10/src/lib/ecore_evas/Makefile.am
===================================================================
--- ecore-1.7.10.orig/src/lib/ecore_evas/Makefile.am	2013-12-18 03:21:05.000000000 +0800
+++ ecore-1.7.10/src/lib/ecore_evas/Makefile.am	2015-12-07 17:17:16.493000656 +0800
@@ -17,7 +17,7 @@
 endif
 
 if BUILD_ECORE_DIRECTFB
-ECORE_DIRECTFB_INC = -I$(top_srcdir)/src/lib/ecore_directfb -I@DIRECTFB_CFLAGS@
+ECORE_DIRECTFB_INC = -I$(top_srcdir)/src/lib/ecore_directfb @DIRECTFB_CFLAGS@
 ECORE_DIRECTFB_LIB = $(top_builddir)/src/lib/ecore_directfb/libecore_directfb.la
 else
 ECORE_DIRECTFB_INC =
Index: ecore-1.7.10/src/lib/ecore_evas/Makefile.in
===================================================================
--- ecore-1.7.10.orig/src/lib/ecore_evas/Makefile.in	2013-12-20 03:52:57.000000000 +0800
+++ ecore-1.7.10/src/lib/ecore_evas/Makefile.in	2015-12-07 17:17:31.697000929 +0800
@@ -602,7 +602,7 @@
 @BUILD_ECORE_FB_FALSE@ECORE_FB_LIB = 
 @BUILD_ECORE_FB_TRUE@ECORE_FB_LIB = $(top_builddir)/src/lib/ecore_fb/libecore_fb.la
 @BUILD_ECORE_DIRECTFB_FALSE@ECORE_DIRECTFB_INC = 
-@BUILD_ECORE_DIRECTFB_TRUE@ECORE_DIRECTFB_INC = -I$(top_srcdir)/src/lib/ecore_directfb -I@DIRECTFB_CFLAGS@
+@BUILD_ECORE_DIRECTFB_TRUE@ECORE_DIRECTFB_INC = -I$(top_srcdir)/src/lib/ecore_directfb @DIRECTFB_CFLAGS@
 @BUILD_ECORE_DIRECTFB_FALSE@ECORE_DIRECTFB_LIB = 
 @BUILD_ECORE_DIRECTFB_TRUE@ECORE_DIRECTFB_LIB = $(top_builddir)/src/lib/ecore_directfb/libecore_directfb.la
 @BUILD_ECORE_WIN32_FALSE@ECORE_WIN32_INC = 
