Index: linux-amlogic-3.10-bringup/include/linux/amlogic/amports/vformat.h
===================================================================
--- linux-amlogic-3.10-bringup.orig/include/linux/amlogic/amports/vformat.h	2014-12-02 10:43:26.000000000 +0800
+++ linux-amlogic-3.10-bringup/include/linux/amlogic/amports/vformat.h	2014-12-02 14:18:39.572616050 +0800
@@ -35,6 +35,7 @@
     VFORMAT_H264MVC,
     VFORMAT_H264_4K2K,
     VFORMAT_HEVC,
+		VFORMAT_UNSUPPORT,
     VFORMAT_MAX
 } vformat_t;
 
